lib/rubber/cloud/digital_ocean.rb

```diff
@@ -115,6 +137,25 @@ def describe_instances(instance_id=nil)
       def active_state
         'active'
       end
+
+      def destroy_instance(instance_id)
+        # The Digital Ocean API will return a 422 if we attempt to destroy an
+        # instance that's in the middle of booting up, so wait until it's
+        # in a non-"new" state
+        print "Waiting for non-new instance state"
```

Comment:
I did the same as create instance in lib/rubber/recipes/rubber/instances.rb, because I was going for a single line statement with periods appended to it as the user waited.  Not a huge deal though, I can change it to a logging statement